### PR TITLE
update webstorm to latest version

### PR DESCRIPTION
**Pull request by Kleak**

### CODE REVIEW

File: programming/webstorm/pspec.xml [hunk 1]
```diff
@@ -10,7 +10,7 @@
         <PartOf>programming</PartOf>
         <Summary xml:lang="en">WebStorm - an IDE for the Web</Summary>
         <Description xml:lang="en">WebStorm - an IDE for the Web</Description>
-        <Archive type="targz" sha1sum="531881bc5760dfaf05e4d8888f89a4a9c3e4f1a9">https://download.jetbrains.com/webstorm/WebStorm-2018.1.3.tar.gz</Archive>
+        <Archive type="targz" sha1sum="22c48fcaecedb0c0f308c24645c8247275b8bf4bb60ac3b3404f092ac92c5d11">https://download.jetbrains.com/webstorm/WebStorm-2018.1.4.tar.gz</Archive>
```
Comment (Staudey) [inline programming/webstorm/pspec.xml]: That's the sha256sum, not the sha1sum. Does this actually work on your end?

Comment (Staudey) [reply programming/webstorm/pspec.xml]: Just for the sake of completeness, the sha1sum would be fefec3422ce2b4a3a67bf1cec86deb6e650ced9d

**Comment by Staudey** [comment]
There is still a small thing missing, see my PR in your repo.
Even with that PR there are three warnings during the creation of the eopkg, but it installs just fine. Not sure how significant those warnings are or how to fix them to be honest.

**Comment by ahahn94** [comment]
Just so you know, I will post my weekly pull request containing all new versions of the jetbrains IDEs later today.

Concerning the warnings during the package build: There have been warnings before I started my batch updates and the number of warnings does not change between versions. They are no problem.

**Pull request closed without merging**